src/js/schemas/job-schema/General.js

```diff
@@ -52,6 +91,22 @@ const General = {
           type: 'number',
           getter(job) {
             return `${job.getDisk() || ''}`;
+          },
+          externalValidator({general}, definition) {
+            if (ValidatorUtil.isEmpty(general.disk)) {
```

Comment:
Please don't use `ValidatorUtil.isEmpty` to check if a property is undefined - we need to refactor that method as the name is highly mislead but is meant to check if an object or array is empty. See DCOS-9453 for details.